json-endpoints.php

```diff
@@ -2570,20 +2914,22 @@
 		'twitter_via'                  => '(string) Twitter username to include in tweets when people share using the Twitter button',
 		'jetpack-twitter-cards-site-tag' => '(string) The Twitter username of the owner of the site\'s domain.',
 		'eventbrite_api_token'         => '(int) The Keyring token ID for an Eventbrite token to associate with the site',
-		'holidaysnow'                  => '(bool) Enable snowfall on front end of site?',
+		'holidaysnow'                  => '(bool) Enable snowfall on frontend of site?',
 		'timezone_string'              => '(string) PHP-compatible timezone string like \'UTC-5\'',
 		'gmt_offset'                   => '(int) Site offset from UTC in hours',
+		Jetpack_SEO_Utils::FRONT_PAGE_META_OPTION => '(string) The seo meta description for the site.',
+		Jetpack_SEO_Titles::TITLE_FORMATS_OPTION => '(array) SEO meta title formats. Allowed keys: front_page, posts, pages, groups, archives',
 		'verification_services_codes'  => '(array) Website verification codes. Allowed keys: google, pinterest, bing, yandex',
-		Jetpack_SEO_Utils::FRONT_PAGE_META_OPTION => '(string) The SEO meta description for the site.',
-		Jetpack_SEO_Titles::TITLE_FORMATS_OPTION  => '(array) SEO meta title formats. Allowed keys: front_page, posts, pages, groups, archives',
-		'api_cache'                    => '(bool) Turn on/off the Jetpack JSON API cache'
+		'amp_is_enabled'               => '(bool) Whether AMP is enabled for this site',
+		'podcasting_archive'           => '(string) The post category, if any, used for publishing podcasts',
+		'api_cache'                    => '(bool) Turn on/off the Jetpack JSON API cache',
```

Comment:
Are we duplicating request fields between v1.1 and v1.2 of this endpoint? Might need to include `site_icon` then.

See: 6c08795cec3cf5e1475beb3152eb99d80c31e122 (11161258e6e4518a74b1b499cc21583a8cc6c057)